src/sage/rings/polynomial/laurent_polynomial_mpair.pyx

```diff
@@ -1155,21 +1156,56 @@ cdef class LaurentPolynomial_mpair(LaurentPolynomial):
         return [a.eadd(self._mon) for a in self._poly.exponents()]
 
     def degree(self, x=None):
-        """
-        Return the degree of ``x`` in ``self``.
+        r"""
+        Return the degree of ``self``.
+
+        INPUT:
+
+        - ``x`` -- (default: ``None``) a generator of the parent ring
+
+        OUTPUT:
+
+        If ``x`` is ``None``, return the maximum degree of the monomials of
+        ``self``.  If ``x`` is given and it is a generator of the parent
+        ring, the output is the maximum degree in ``x`` of ``self``.
```

Comment:
```suggestion
        If ``x`` is ``None``, return the total degree of ``self``.
        If ``x`` is a given generator of the parent ring,
        the output is the maximum degree of ``x`` in ``self``.
```
I think this is a better way to write this. Total degree is a very commonly defined concept IMO, so there will be no ambiguity. Maximum degree of the monomials is vague to the wrong definition (depending on the interpretation).